packages/bits/src/lib/select-v2/base-select-v2.ts

```diff
@@ -387,7 +387,9 @@ export abstract class BaseSelectV2 implements AfterViewInit, AfterContentInit, C
     }
 
     private setActiveItemOnDropdown(): void {
-        this.value && !this.multiselect
+        const selectedValue = this.options?.find(option => isEqual(option.value, this.value));
```

Comment:
`this.value` may be one option or an array of options in the case of multi select. It seems like we need to handle multi select here as well.